metaflow/metaflow_config.py

```diff
@@ -433,6 +433,12 @@
 # should result in an appreciable speedup in flow environment initialization.
 CONDA_DEPENDENCY_RESOLVER = from_conf("CONDA_DEPENDENCY_RESOLVER", "conda")
 
+# Conda Fast init binary url
+CONDA_FAST_INIT_BIN_URL = from_conf("CONDA_FAST_INIT_BINARY_URL", None)
```

Comment:
can you remove this config and introduce the path to actual binary in the `install_fast_initializer` method instead? also, if you can note some latency improvements in the PR, that will be great!